test/controllers/groups_controller_test.rb

```diff
@@ -76,7 +76,7 @@ class GroupsControllerTest < ActionDispatch::IntegrationTest
   test 'should delete a group' do
     sign_in users(:john_doe)
 
-    group = groups(:group_one)
+    group = groups(:group_two)
```

Comment:
Can you also add a test here and the test files below to use `group_one` as well to ensure it deletes the group and all it's subgroups

Reply:
Awesome great idea. Thanks!